@navikt/ds-react/grid/src/Cell.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { forwardRef, HTMLAttributes } from "react";
+import cl from "classnames";
+import "@navikt/ds-css/grid/index.css";
+
+type Column = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
+export interface CellProps extends HTMLAttributes<HTMLDivElement> {
+  children?: React.ReactNode;
+  className?: string;
+  white?: boolean;
+  padding?: boolean;
+  sm: Column;
+  md?: Column;
+  lg?: Column;
```

Comment:
sm, md, lg og xl kommer fra Boostrap, eller? Jeg synes de er litt kryptiske. Jeg liker "small", "medium", "large", og "xl", som her: https://github.com/navikt/nav-frontend-moduler/blob/master/@navikt/ds-react/typography/src/Heading.tsx#L6

Reply:
Her er jeg også veldig åpen for å overbevises.

Reply:
Tja, personlig syntes jeg "sm", "md", "lg" og "xl" hører mer sammen enn "small", "medium", "large" og "xl". Følter vi kombinerer fulle ord med forkortelser. Men det er viktigere at det er likt på tvers av komponentene, så jeg endrer til det dere bestemmer :) Tanker?

Reply:
Ja, helt enig i at å bestemme en standard er viktigst. Hva liker du, @KenAJoh ?

Reply:
Jeg titta litt rundt, og det er mange ulike varianter der ute, men sm, md, lg, xl er nok vanligst. Jeg fant faktisk ingen som gjør det sånn som jeg foreslo. Det jeg heller mot nå er sånn som Shopify gjør det her: https://polaris.shopify.com/design/typography. De bruker small, medium, large, x-large. Jeg liker det best fordi det krever null tolking fra den som leser.

Reply:
Den er god :) Fikser